tensorflow/core/BUILD

```diff
@@ -1348,9 +1380,8 @@ tf_cuda_library(
         ":lib",
         ":lib_internal",
         ":protos_all_cc",
-        ":stream_executor",
         "//third_party/eigen3",
-    ],
+    ] + if_not_windows([":stream_executor"]),
```

Comment:
I found there are  4 targets we need to modify if we want to cut the dependency on `gpu_runtime`.
`//tensorflow/core:core`, 
`//tensorflow/core:tensorflow_opensource`, 
`//tensorflow/core/util/ctc:ctc_beam_search_lib`, 
`//tensorflow/core/kernels:array`
As you can see, they are scattered in 3 different files, I kind of prefer the original change, so it will be easier when we enable gpu support later. WDYT?